crates/nargo_cli/src/cli/mod.rs

```diff
@@ -49,6 +49,7 @@ enum NargoCommand {
     Check(check_cmd::CheckCommand),
     CodegenVerifier(codegen_verifier_cmd::CodegenVerifierCommand),
     Compile(compile_cmd::CompileCommand),
+    Build(compile_cmd::CompileCommand),
```

Comment:
```suggestion
    #[command(alias = "build")]
    Compile(compile_cmd::CompileCommand),
```

We can [create aliases in clap ](https://docs.rs/clap/latest/clap/struct.Command.html#method.alias) which would enforce that `build` and `compile` are handled the same.